custom_dc/faq.md

```diff
@@ -19,9 +19,12 @@ Yes; there are many options for doing so. If you want an entirely private site w
 
 Note that you cannot apply fine-grained access restrictions, such as access to specific data or pages. Access is either all or nothing. If you want to be able to partition off data, you would need to create additional custom instances.
 
-### Will my custom data end up in the base Data Commons?
+### Will my data or queries end up in base Data Commons? {#data-security}
 
-No. Even if a query needs data from the base Data Commons, gleaned from the (incomplete) results from your custom data, it will never transfer any custom data to the base data store.
+Your user queries, observations data, or property values are never transferred to base Data Commons. The NL model built from your custom data lives solely in your custom instance. The custom Data Commons Mixer does make API calls to the base Data Commons Mixer (as depicted in [this diagram](/custom_dc/index.html#system-overview)) only in the following instances:
```

Comment:
Have we introduced them to mixer anywhere in the document? If not, may be we just say that "The custom Data Commons _instance_ does make API calls to base Data Commons..."

Reply:
Yes, right here: https://screenshot.googleplex.com/BoJEbJzgQZhG4vZ. However, you're right that it's simpler to just say "instance".